l10n_it_fatturapa_sale/tests/test_fatturapa_sale.py

```diff
@@ -43,16 +46,26 @@ def test_create_invoice(self):
         order_line, sale_order = self._create_order()
 
         # Check the invoice
-        invoice_ids = sale_order.action_invoice_create()
```

Comment:
Così però viene rimosso il test del flusso precedente, è possibile testare il nuovo flusso in un test dedicato?